modules/ocf_desktop/manifests/xsession.pp

```diff
@@ -59,14 +59,19 @@
   }
 
   # wallpaper symlink
-  $wallpaper = $staff_only ? {
-    true  => 'background-staff.svg',
-    false => 'background.svg',
+  $login_screen = $staff_only ? {
+    true    => 'login-staff.png',
+    default => 'login.png',
   }
 
   file { '/opt/share/wallpaper':
     ensure  => link,
-    target  => "/opt/share/xsession/images/${wallpaper}",
+    target  => '/opt/share/xsession/images/background.png',
```

Comment:
Isn't this file getting removed? Should this link be updated, or should the file not be removed?

Reply:
It's not getting removed. `background.png` is the background once you're logged in.